Marlin/temperature.cpp

```diff
@@ -614,6 +668,12 @@ void manage_heater() {
       thermal_runaway_protection(&thermal_runaway_state_machine[e], &thermal_runaway_timer[e], current_temperature[e], target_temperature[e], e, THERMAL_PROTECTION_PERIOD, THERMAL_PROTECTION_HYSTERESIS);
     #endif
 
+    #if MAX_TEMP_OVERSTOOT_TIME > 0
+      for (int8_t e = -1; e < EXTRUDERS; e++) {
```

Comment:
@thinkyhead 
On the long term, beginning the arrays with the bed as 0 , and seeing as a heater as all the others, will save a lot of code in `temperature.h`.